features/admin_creates_auction.feature

```diff
@@ -18,6 +18,7 @@ Feature: Admin creates auctions in the admins panel
     Then I should see the start time I set for the auction
     And I should see the end time I set for the auction
 
+  @freeze_time
```

Comment:
One trick might be to litter the code path with some `puts` statements, and change the `@freeze_time` date by a day and find the precise day that begins to cause the error. Then compare the `puts` output.